app/services/npq/application/accept.rb

```diff
@@ -145,6 +151,28 @@ def validate_permitted_schedule_for_course
           errors.add(:schedule_identifier, I18n.t(:schedule_invalid_for_course))
         end
       end
+
+      def eligible_for_funded_place
+        return unless FeatureFlag.active?("npq_capping")
+        return if errors.any?
+        return if funded_place.nil?
+
+        unless npq_application.eligible_for_funding && funded_place
+          errors.add(:npq_application, I18n.t("npq_application.not_eligible_for_funded_place"))
+        end
+      end
+
+      def funding_cap_available
+        return unless FeatureFlag.active?("npq_capping")
+        return if errors.any?
+        return if funded_place.nil?
+
+        contract = npq_application.current_contract
+
+        return if funded_place && contract.funding_cap.present? && contract.funding_cap.positive?
+
+        errors.add(:npq_application, I18n.t("npq_application.no_funding_cap_available"))
```

Comment:
I think maybe the check here will probably be that this field is required since there is a cap on the contract 🤔 otherwise it's ok?